packages/flutter_tools/lib/src/dart/summary.dart

```diff
@@ -74,3 +78,57 @@ void buildSkyEngineSdkSummary(
   String outputPath = pathos.join(skyEnginePath, outBundleName);
   new io.File(outputPath).writeAsBytesSync(output.sum);
 }
+
+Future<Null> buildUnlinkedForPackages(String flutterPath) async {
+  PhysicalResourceProvider provider = PhysicalResourceProvider.INSTANCE;
+  PubSummaryManager manager =
+      new PubSummaryManager(provider, '__unlinked_${io.pid}.ds');
+
+  Folder flutterFolder = provider.getFolder(flutterPath);
+
+  //
+  // Build in packages/.
+  //
+  Folder packagesFolder = flutterFolder.getChildAssumingFolder('packages');
+  await _buildUnlinkedForPackage(
```

Comment:
When wrapping lines like this, please pick one of the following styles (in order of preference, preferred first):

```dart
await _buildUnlinkedForPackage(
  manager,
  packagesFolder.getChildAssumingFolder('flutter')
);
```

```dart
await _buildUnlinkedForPackage(
  manager, packagesFolder.getChildAssumingFolder('flutter')
);
```

```dart
await _buildUnlinkedForPackage(manager,
    packagesFolder.getChildAssumingFolder('flutter'));
```

```dart
await _buildUnlinkedForPackage(manager, packagesFolder.getChildAssumingFolder('flutter'));
```